### ISSUE TITLE
attribute-string-redundant: false positive on related fields

### ISSUE THREAD

**Issue by luisg123v** [OCA/pylint-odoo#332]
When a related field is declared, the `string` attribute is taken by default from the referenced field, so redefining the string is a valid case. For instance:

```
parent_date = fields.Date(
    related="parent_id.date",
    string="parent date",
)
```

If a string is not specified, field will be "Date".
But if I need the string to be "Parent Date", the lint is triggered.

**Comment by luisg123v** [comment]
CC @hbto @moylop260 @fernandahf
